src/layout/drawer/nav/index.tsx

```diff
@@ -0,0 +1,41 @@
+import LogoutIcon from '@mui/icons-material/Logout'
+import { ListItemButton, ListItemText, ListItemIcon } from '@mui/material'
+import List from '@mui/material/List'
+import React from 'react'
+import styled from 'styled-components'
+
+import { LayoutDrawerNavConfig } from './config'
+import LayoutDrawerNavItem from './item'
+import { ListItemProps } from './types'
+
+const LayoutDrawerNav: React.FC = () => {
+  return (
+    <>
+      <List>
+        {LayoutDrawerNavConfig.map((item: ListItemProps) => (
+          <LayoutDrawerNavItem key={item.name} {...item} />
+        ))}
+      </List>
+      <StyledList>
+        <StyledListItemButton>
+          <ListItemIcon>
+            <LogoutIcon />
+          </ListItemIcon>
+          <ListItemText primary="Logout" />
+        </StyledListItemButton>
+      </StyledList>
```

Comment:
Niepotrzebne to jest, może przeniesiemy ten Logout button do `LayoutDrawer` 
W sumie to nie musimy używać `ListItemButton` tylko zwykły `Button`.
W tym samym pliku możemy na samą górę wrzucić `logo`